lib/mandrill_dm/message.rb

```diff
@@ -248,6 +258,10 @@ def return_string_value(field)
       mail[field] ? mail[field].to_s : nil
     end
 
+    def return_time_parsed_string_value(field)
+      mail[field] ? Time.parse(mail[field].value).utc.strftime("%Y-%m-%d %H:%M:%S") : nil
```

Comment:
@kshnurov yup, you're definitely right. sorry about that, what can I say. hopefully this will get us closer here